fellow-submissions/Paul-Sobers/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Mansalva&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css" type="text/css" />
+
+  <title>Document</title>
+</head>
+<body>
+<header>
+  <h1>Welcome To Paul Gaming quiz</h1>
+  <p>In this quiz you will answer 5 questions about my favorite games.</p>
+  <img src="https://singularityhub.com/wp-content/uploads/2018/11/young-gamer-playing-video-game_shutterstock_624801452-1068x601.jpg">
+</header>
+<form action ="#" method="post">
+   <fieldset>
+    <fieldset class="game-name">
+      <ul>
+        <li>
+    <label for ="game">1: What is your fav COD?</label>
+    <input type="text" name="game" placeholder="COD" /> 
+     </li>
+  </ul>
+  </fieldset>
+  <fieldset class="game-list">
+    <ul>
+      <li>
+        <label for="gamelist">2: What game on this list has a battle royale mode?</label>
+        <select name="gamelist">
+          <option value="Fortnite">Fortnite</option>
+          <option value="MW4">Modern Warfare Alpha</option>
+          <option value="NBA2k">NBA 2k</option>
+          <option value="Candy Crush">Candy Crush</option>
+          <option value="Super" selected>Super Smash Bros</option>
+        </select>
+      </li>
+    </ul>
+  </fieldset>
+<fieldset class="days">
+  <ul>
+    <li>
+      <label for="game-day">3: What days of the week do you think I play the most games on ? Check all that apply:</label>
+      <input type="checkbox" name="game-day" value="Sunday" id="game-day" checked>Sunday
+      <input type="checkbox" name="game-day" value="Monday" id="game-day">Monday
+      <input type="checkbox" name="game-day" value="Tuesday" id="game-day">Tuesday
+      <input type="checkbox" name="game-day" value="Wednesday" id="game-day">Wednesday
+      <input type="checkbox" name="game-day" value="Thursday" id="game-day">Thursday
+      <input type="checkbox" name="game-day" value="Friday" id="game-day">Friday
+      <input type="checkbox" name="game-day" value="Saturday" id="game-day">Saturday
```

Comment:
You know, Paul, that each `id` should be unique. Your instincts are right here. `id`s would be nice here because we could associate our `<input>`s with our `<label>s` but here's how you implement it:
```html
<input type="checkbox" name="game-day" value="Sunday" id="sunday">
<label for="sunday">Sunday</label>
```

In sum, your `for` attribute on your `<label>` tag should match the `id` on your `<input>`. And most importantly, each one of these checkboxes needs its own label.